tsl/src/continuous_aggs/create.c

```diff
@@ -784,6 +784,13 @@ cagg_validate_query(Query *query)
 		Dimension *part_dimension = NULL;
 		ht = ts_hypertable_cache_get_cache_and_entry(rte->relid, CACHE_FLAG_NONE, &hcache);
 
+		if (hypertable_is_distributed(ht))
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg("hypertable is a distributed hypertable"),
+					 errhint("creating continuous aggregates for distributed hypertables is not "
```

Comment:
I would use errdetail here since hint is usually about how to resolve/fix the error.  But with my suggested change to main error this detail seem a bit redundant.